src/rider/main/kotlin/me/seclerp/rider/plugins/efcore/commands/CommonOptionsMapper.kt

```diff
@@ -0,0 +1,12 @@
+package me.seclerp.rider.plugins.efcore.commands
+
+class CommonOptionsMapper {
```

Comment:
Let's make it on the backend side. We don't need to know raw target framework name like `.NETCoreApp,Version=v3.1` on frontend side

Reply:
And also it will be more intuitive if we wrap all cases in switch:
```csharp
=> framework switch
{
  ".NETCoreApp,Version=v3.1" => "netcoreapp3.1",
  ".NETStandard,Version=v2.1" => "netstandard2.1",
  _ => framework 
}
```
And I think that it's better to make it as an extension method for `TargetFrameworkId` type